css-two-column-layout/styles.css

```diff
@@ -0,0 +1,68 @@
+h3 {
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+h1 {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 36px;
+}
+
+.header {
+  text-align: center;
+  padding-left: 16%;
+  padding-right: 16%;
+}
+
+body {
+  font-family: karla, sans-serif;
+}
+
+.container {
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  flex-wrap: nowrap;
+  height: 160px;
+  padding-top: 50px;
+}
+
+.four-fifths {
+  padding-right: 5%;
+  padding-top: 20px;
+}
+
+button {
+  color: rgb(255 81 0 / 86.5%);
+  border-color: rgb(255 81 0 / 86.5%);
+  font-size: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 30px;
+  display: block;
+  margin-left: 325px;
+  margin-top: 35px;
+}
+
+p {
+  line-height: 2;
+  font-size: 12px;
+}
+
+.one-half {
```

Comment:
make sure to have a proper flex basis for this class. We always want to explicitly set a column width